src/h3/mod.rs

```diff
@@ -637,6 +637,20 @@ impl Connection {
         &mut self, conn: &mut super::Connection, stream_id: u64,
         headers: &[Header], fin: bool,
     ) -> Result<()> {
+        match self.streams.get_mut(&stream_id) {
+            Some(s) => {
+                conn.stream_priority(
+                    stream_id,
+                    s.priority.urgency,
```

Comment:
Do we need to map Priority header value to quiche stream priority? It looks like Priority header urgency default is 1 but stream priority default is 127 (https://github.com/cloudflare/quiche/blob/57d448cb6bf28cc5feba347c9fb99d418b26125a/src/stream.rs#L41)

Reply:
The spaces are different so have different rules. quiche's default is "halfway" in the space while HTTP priority default is "close to most urgent" because most HTTP resources will fit that bucket and it means that no header needs to be sent.

In the HTTP/3 application mapping, no stream should ever operate at quiche's default. 

In order to make the code flexible and robust, we should create a conversion methods between the two spaces.